### DISCUSSION TITLE
submission error

### DISCUSSION THREAD

**Discussion by bingblackbean** [PaddlePaddle/PaddleSpatial#73] [BaiduKDDCup2022] [answered]
Dear Organizer,
I got an error "`operands could not be broadcast together with shapes (288,) (1856,)`" after the submission, but my_tests have been test in my local env.

could you help to provide any hint about this?

I checked the `predict.py` in` test-1.zip` folder and find `forecast_one` returns:
`  prediction = prediction[0]`
could you explain this change?

**Comment by bingblackbean** [comment, marked as answer]
problem is solved, please close it

**Reply by Paul-Cui** [reply]
Can you share the answer? I have the same problem.

**Reply by bingblackbean** [reply]
> Can you share the answer? I have the same problem.

unfortunately not, I have no clue about why this error pops up.   
The "solution" is that I give up my own code and use the baseline code (switch to the pytorch version) to submit.
The most weird thing was that my own code can pass the test in my local env with the toy data and get the final score.


It is hard for me to debug this error now since my code is entirely changed.

**Reply by Paul-Cui** [reply]
Got it，  Thanks a lot！
